readme.md

```diff
@@ -0,0 +1,243 @@
+# remark-breaks
+
+[![Build][build-badge]][build]
+[![Coverage][coverage-badge]][coverage]
+[![Downloads][downloads-badge]][downloads]
+[![Size][size-badge]][size]
+[![Sponsors][sponsors-badge]][collective]
+[![Backers][backers-badge]][collective]
+[![Chat][chat-badge]][chat]
+
+[**mdast**][mdast] utility to support hard breaks without needing spaces or
```

Comment:
```suggestion
**[mdast][]** utility to support hard breaks without needing spaces or
```

I now prefer putting the attention (emphasis, strong) outside the link, as that allows for using a collapsed reference